src/Microsoft.OpenApi.Readers/V2/OpenApiResponseDeserializer.cs

```diff
@@ -85,20 +91,58 @@ private static void ProcessProduces(MapNode mapNode, OpenApiResponse response, P
                 {
                     var mediaType = new OpenApiMediaType
                     {
-                        Schema = schema
+                        Schema = schema,
+                        Examples = examples
                     };
 
                     response.Content.Add(produce, mediaType);
                 }
             }
 
             context.SetTempStorage(TempStorageKeys.ResponseSchema, null, response);
+            context.SetTempStorage(TempStorageKeys.Examples, null, response);
             context.SetTempStorage(TempStorageKeys.ResponseProducesSet, true, response);
         }
 
+        private static void LoadExamplesExtension(OpenApiResponse response, ParseNode node)
+        {
+            var mapNode = node.CheckMapNode("x-examples");
+            var examples = new Dictionary<string, OpenApiExample>();
+
+            foreach (var examplesNode in mapNode)
+            {
+                // Load the media type node as an OpenApiExample object
+                var example = new OpenApiExample();
+                var exampleNode = examplesNode.Value.CheckMapNode(examplesNode.Name);
+                foreach (var valueNode in exampleNode)
+                {
+                    switch (valueNode.Name)
```

Comment:
do we need to normalize casing here?

Reply:
I don't think its necessary but I can add it just in case

Reply:
where are those keys coming from? are they part of the schema or extension keys?

Reply:
Extension keys